libsolidity/ast/ASTAnnotations.h

```diff
@@ -245,14 +246,14 @@ struct ExpressionAnnotation: ASTAnnotation
 	/// Whether the expression is a constant variable
 	bool isConstant = false;
```

Comment:
Why not this one?

Reply:
If I recall correctly, it was quite cumbersome setting this in all the visitors correctly. I can have another look though.

Reply:
Ah right, I remember. It's fine!